ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py

```diff
@@ -58,18 +58,18 @@ def generate_launch_description():
     diff_drive_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
+        arguments=["diff_cont"],
```

Comment:
Can we leave these timeouts in? For some reason my computer needs the increased timeout
```suggestion
        arguments=["diff_cont"], "--controller-manager-timeout", "30"],
```

Reply:
Yeah, I don't see a problem with that.